utils/logger.js

```diff
@@ -0,0 +1,109 @@
+const createLogger = () => {
+  const isProdMode =
+    process.env.NODE_ENV === "prod" || process.env.NODE_ENV !== "development"
+      ? true
+      : false;
+
+  //TODO: move this var to env file.
+  // level 1 = debug
+  // level 2 = default
+  // level 3 = no logging
+  let logLevel = 2;
+
+  if (isProdMode) {
+    return createProdLogger(logLevel);
+  }
+
+  return createDevLogger(logLevel);
+};
+
+const createProdLogger = (logLevel) => {
+  //TODO: read log destination from env. (QA/Prod might be different destination, but log the same)
+  //TODO: implement a real Prod logger
+
+  const debugLogFunction = (message) => {
+    if (logLevel > 1) return;
+
+    const currentTime = new Date().toLocaleTimeString();
+    console.log(`${currentTime}: ${message}`);
```

Comment:
If we're going through the trouble of setting up a specific logger framework, shouldn't we log to a file instead of console?

Reply:
I just wanted to make the abstraction logger in this PR. 
I left as console for now and made a separate issue for implementing a real logger

Reply:
Gotcha, good deal.